Toolkit/ArcGISToolkit/TimeSlider.swift

```diff
@@ -185,7 +185,7 @@ public class TimeSlider: UIControl {
     }
     
     /**
-     A Boolean value that indicates whether the start time of the currentExtent can be
+     A boolean value that indicates whether the start time of the currentExtent can be
```

Comment:
Not sure what prompted this change, but Boolean is capitalized in the New Oxford American dictionary and in all of Apple's API documentation.

Reply:
Yes, you are right. I did not check the dictionary. Fixed.